docs/api/session.md

```diff
@@ -831,6 +831,21 @@ this session just before normal `preload` scripts run.
 Returns `String[]` an array of paths to preload scripts that have been
 registered.
 
+#### `ses.setCodeCachePath(path)`
+
+* `path` String - The Code cache location.
+
+Sets code cache directory. By default, the directory will be `Code Cache` under the
```

Comment:
The document should make it clear this is for generated code from the renderer, only "code cache" is a bit vague.